docs/variants/msi_z690/community/community-test-results.md

```diff
@@ -0,0 +1,32 @@
+# Community test results - MSI Z690-A WIFI DDR4
+
+## About
+
+The following document contains information about additional tests which were
+carried out on the MSI Z690-A WIFI DDR4 platform with Dasharo firmware by
+memmbers of the community.
+
+The following list is created based on the information from the
+community - people which are currently using the platform with Dasharo
+firmware.
+
+This document contains three main sections:
+- `Testing method` in which we are describing our preferred testing method.
+- `Hardware Compatible List` in which in a labelled manner we show all
+    tests carried out by the community.
+- `Support us` - in which we describe the method of how it is possible to
+    contribute to the development of this list.
+
+## Testing method
+
+## Community test results
+
+| No.  | Supported hardware                                | Link to description | Notes               |
```

Comment:
This table is identical to the HCL table. Is this intentional? What is the difference between these?
Maybe we should include some examples at least.

We do not propose any format, so we will get a plethora of possible outputs, which would be very difficult to put in there.